assets/stylesheets/style.scss

```diff
@@ -23,15 +23,11 @@
 	//@import 'shared/welcome'; // welcome messages
 	//@import 'shared/infinite-scroll-end'; // Last page marker once infinite scroll has reached end
 
-	@import 'shipping-services'; // Shipping Services Groups, Group and Entry Component styles
-	@import 'shipping-label'; // Shipping Labels specific styles
-
 	// Main
 	@import 'main'; // global layout and responsive styles
 
 	// Components
 	@import 'components';
-	@import 'components/payment-logo/style';
 
 	// Breakpoints
 	$breakpoints: 480px, 660px, 780px, 960px, 1040px; // adds wp-admin specific breakpoints
```

Comment:
Calypso doens't use 780 - let's remove our uses of it. [Calypso's breakpoints](https://github.com/Automattic/wp-calypso/blob/master/assets/stylesheets/shared/mixins/_breakpoints.scss#L6) are:

`$breakpoints: 480px, 660px, 960px, 1040px, 1280px, 1600px;`